flask_rest_orm/resources/utils.py

```diff
@@ -75,16 +75,20 @@ def query_from_request(model, request):
     """
     query = model.query
 
-    def build_query(column, request_filter):
+    def build_filter_operator(column_name, request_filter):
```

Comment:
Could you add a documentation with some examples of filters? A simple one: `{'col1': 'text'}`, one using SQLAlchemy operators and another using the $or, $and operators.